src/main/java/LacksDataBaseProject/AddAndRemoveLack.java

```diff
@@ -0,0 +1,53 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveLack extends CheckUserRole {
+    static Scanner input = new Scanner(System.in);
+
+    protected void addlack(Lack lack, User user, Supplier supplier) {
+        if (checkIfForwarder(user)) {
```

Comment:
Tutaj moim zdaniem dokładnie to co pisałem wyżej, na dobrą sprawę powinniscie przekazać do metody gotowy lack z wypełnionymi polami tak by nie korzystać z scannera :)